src/multivariate/mvnormal.jl

```diff
@@ -80,6 +80,27 @@ abstract type AbstractMvNormal <: ContinuousMultivariateDistribution end
 insupport(d::AbstractMvNormal, x::AbstractVector) =
     length(d) == length(x) && all(isfinite, x)
 
+"""
+    minimum(d::AbstractMvNormal)
+
+Return the minimum of the support of each dimension of `d`.
+"""
+minimum(d::AbstractMvNormal) = fill(eltype(d)(-Inf), length(d))
+
+"""
+    maximum(d::AbstractMvNormal)
+
+Return the maximum of the support of each dimension of `d`.
+"""
+maximum(d::AbstractMvNormal) = fill(eltype(d)(Inf), length(d))
+
+"""
+    extrema(d::AbstractMvNormal)
+
+Return the minimum and maximum of the support of each dimenison of `d` as a 2-tuple.
+"""
+extrema(d::AbstractMvNormal) = minimum(d), maximum(d)
+
```

Comment:
It is a bit unfortunate that one has to define this (the default implementation in Julia falls back to iterating over the argument which is not supported by distributions). I guess it would be reasonable to only define
```julia
Base.extrema(d::Distribution) = minimum(d), maximum(d)
```
once in Distributions?

Reply:
👍

Reply:
agreed... I was actually surprised it didn't have a fallback when I implemented it